README.md

```diff
@@ -1,21 +1,61 @@
-# LTI 1.3 Demo
+# LTI 1.3 Demo Application
 
-> [Symfony](https://symfony.com/) based demo application for LTI 1.3.
+> [Symfony](https://symfony.com/) demo application for LTI 1.3, to act as a [LTI Platform and Tool](http://www.imsglobal.org/spec/lti/v1p3/#platforms-and-tools-0).
+
+This demo application based on:
+- the [lti1p3 symfony bundle](https://github.com/oat-sa/bundle-lti1p3)
+- the [lti1p3-core library](https://github.com/oat-sa/lib-lti1p3-core)
 
 ## Table of Contents
 
+- [Specifications](#specifications)
 - [Installation](#installation)
 - [Usage](#usage)
 
+## Specifications
+ 
+ This demo application respect the following LTI 1.3 [IMS](http://www.imsglobal.org) specifications:
+- [IMS LTI 1.3 Core](http://www.imsglobal.org/spec/lti/v1p3)
+- [IMS Security](https://www.imsglobal.org/spec/security/v1p0)
+
 ## Installation
 
+You need to ensure first you have the following installed:
+- [docker](https://docs.docker.com/get-docker/)
+- [docker-compose](https://docs.docker.com/compose/install/)
+
+After cloning this repository, you can build the [provided docker stack](docker-compose.yml):
 ```console
-$ composer install
+$ docker-compose up -d
 ```
+
+Then, install required dependencies with [composer](https://hub.docker.com/_/composer):
+```console
+$ docker run --rm --interactive --tty \
+  --volume $PWD:/app \
```

Comment:
Just a side note, that on Windows it should be `--volume %cd%:/app`.

Reply:
will add it, the Windows population may be important customer side